src/routes/order.router.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express')
+
+const orderController = require('../controllers/order.controllers')
+
+const orderRouter = express.Router();
+
+orderRouter.post("/orderdetail" , orderController.postOrder);
```

Comment:
```suggestion
orderRouter.post("" , orderController.postOrder);
```
`http -v POST localhost:3000/orders  ~~~~~` RESTful API의 개념에 따라 `what` 과 `how`는 `/orders`라는 엔드포인트에 `POST`하는 것만으로도 충분히 전달이 될 것 같습니다!